### PR TITLE
Pyvmomi

### PR DESCRIPTION
**Pull request by sy-hbk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pg-hbk** [comment]
I'm ok with being a maintainer.

File: recipes/pyvmomi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+ requirements:
+  host:
+    - pip
+    - python
+    - requests >=2.3.0
```
Comment (xhochy) [inline recipes/pyvmomi/meta.yaml]: Is `requests` and `six` necessary to build this packages?

File: recipes/pyvmomi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 653f978fd14474a1d3ffc22b1fdffa8a406987ada4ef6d13d96a37ef4b038891
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (xhochy) [inline recipes/pyvmomi/meta.yaml]: `--no-deps --ignore-installed` are included by default
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

File: recipes/pyvmomi/meta.yaml (hunk 1 above)
Comment (sy-hbk) [reply recipes/pyvmomi/meta.yaml]: Removed `requests` and `six` from host

**Pull request merged**